src/chrome/content/rules/Enhancedsteam.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Enhancedsteam.com">
```

Comment:
Mismatch:
https://translation.enhancedsteam.com/

You should add:
https://api.enhancedsteam.com/
https://firefox.enhancedsteam.com/